packages/@react-spectrum/dialog/src/Dialog.tsx

```diff
@@ -51,7 +51,7 @@ function Dialog(props: SpectrumDialogProps, ref: DOMRef) {
   let formatMessage = useMessageFormatter(intlMessages);
   let {styleProps} = useStyleProps(otherProps);
 
-  size = type === 'popover' ? 'S' : (size || 'L');
+  size = type === 'popover' ? (size || 'S') : (size || 'L');
```

Comment:
popover dialogs supports size now?

Reply:
Yeah, we needed it for the docs so the popovers for type details are so small